internal/scorecard/scorecard.go

```diff
@@ -49,57 +43,62 @@ const (
 
 // make a global logger for scorecard
 var (
-	logReadWriter io.ReadWriter
-	log           = logrus.New()
+	LogReadWriter io.ReadWriter
+	Log           = logrus.New()
 )
 
-var scViper *viper.Viper
+type Config struct {
+	ListOpt         bool
+	OutputFormatOpt string
+	VersionOpt      string
+	ConfigOpt       string
+	SelectorOpt     string
+	Selector        labels.Selector
+	BundleOpt       string
+	Kubeconfig      string
+	Something       string
+	Plugins         []Plugin
+	Configs         []PluginConfig
```

Comment:
`PluginConfigs` would be better so we don't have `Config.Configs`, which is ambiguous.

Reply:
good suggestion, I'll rename it in the next commit.